test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -64,7 +64,15 @@ export function setup() {
 
 			await app.workbench.sqlNotebook.notebookToolbar.changeKernel('Python 3');
 			await app.workbench.configurePythonDialog.waitForConfigurePythonDialog();
-			await app.workbench.configurePythonDialog.installPython();
+			try {
+				await app.workbench.configurePythonDialog.waitForPageOneLoaded();
+			} catch (e) {
+				await app.captureScreenshot('Configure Python Dialog page one not loaded');
```

Comment:
I think it'd be useful to put this into a generic function - something like `takeScreenshotOnFailure(() => { ... })`

Reply:
Will add in a future PR if we need screenshots in more places.